components/Header.js

```diff
@@ -40,6 +40,12 @@ const Header = (props) => (
                         <li className={props.page === 'queue' ? 'active' : null}><Link href="/queue"><a>Queue</a></Link></li>
                     </ul>
                     <ul className="nav navbar-nav navbar-right">
+                        <li>
```

Comment:
There is a small scope of improvement here, currently the octacat icon is being used for both stars and forks.
![Screenshot from 2020-03-03 23-50-05](https://user-images.githubusercontent.com/23582438/75806568-e9b48700-5da9-11ea-8906-e215420a3755.png)
Most of the open sourced projects use different logos for this purpose, for example : 
![Screenshot from 2020-03-03 23-50-22](https://user-images.githubusercontent.com/23582438/75806640-15377180-5daa-11ea-93c3-0b2ddca95ae6.png)
It will be better to have the relevant icons here.

Reply:
Agreed. Thanks for pointing out, @rkpattnaik780!

Reply:
@rkpattnaik780 @coderwassananmol Since there is no official GitHub plugin for the buttons,
I used the official GitHub icons pack to import the icons, and used bootstrap to create the buttons:

![Screenshot from 2020-03-04 15-00-18](https://user-images.githubusercontent.com/42573842/75865038-e7ddd880-5e28-11ea-8f89-90b7cb3da270.png)

Reply:
Looks good, just one small tweak, do show the no.of stars and forks of the repo. I have attached a sample where I have used a bootstrap badge.
![Screenshot from 2020-03-04 15-28-08](https://user-images.githubusercontent.com/23582438/75867877-3f7e4300-5e2d-11ea-988b-dc409ebc7ae2.png)

To get the no. of stars and forks, you may hit this API endpoint https://api.github.com/repos/coderwassananmol/BUB2.

Reply:
I have made the necessary changes. Please review.